skia-bindings/build_support/skia_bindgen.rs

```diff
@@ -715,6 +718,8 @@ pub(crate) mod rewrite {
 
 pub use definitions::{Definition, Definitions};
 
+use super::cargo::Target;
```

Comment:
This `use` belongs to the top of the file. I am assuming that the IDE misplaced it because of the `pub use` before, which also should not belong here..

Reply:
Oops yes, I'll point fingers to rust-analyser ;)